tests/account_tests.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+import requests
+from atmsimulator.account import *
+from unittest.mock import patch
+
+class TestAccount(unittest.TestCase):
+    def setUp(self) -> None:
+        from atmsimulator import globals
+        globals.account={
```

Comment:
`globals.account` within `globals.py` isn't an array of accounts, it's only the account object that is retrieved from the server. Fix your tests and functions within `account.py` accordingly. Apart from that, if the tests work as they're expected to should be all good.